pylhc_submitter/sixdesk_tools/extract_data_from_db.py

```diff
@@ -0,0 +1,99 @@
+"""
+Extract Data From DataBase
+-----------------------------
+
+These functions operate on the SixDesk database and help to extract data
+form it.
+
+TODO: Implement extraction of data into ``.csv`` (and/or tfs?)
```

Comment:
still open or done with `extract_da_data`?

Reply:
still open. that's more of an extra feature which I have not needed yet, but would make sense to have in the future, as Frederik saves the results this way in an easily readable format on EOS (next to the slightly harder readable `.db`).

Reply:
Currently he does this with a bash-script. But as everything is there in python as well, it would be a nice extra function in this module.